src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorage/AzureTableStorageSink.cs

```diff
@@ -39,20 +38,6 @@ public class AzureTableStorageSink : ILogEventSink
         /// <param name="storageAccount">The Cloud Storage Account to use to insert the log entries to.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <param name="storageTableName">Table name that log entries will be written to. Note: Optional, setting this may impact performance</param>
-        public AzureTableStorageSink(
```

Comment:
Unfortunately the binder will still be picky when the assembly is loaded (only the compiler knows about default params). I think it's good to give it the chop :-) but it'd need to be part of a major version number increment (i.e. 4.0.0). Since there are other method signature changes in the PR (same issue) we should probably bump project.json up to 4.x 👍